tests/benchmarks/multicore/idle_pwm_loopback/Kconfig

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config TEST_SLEEP_DURATION_MS
+	int "Amount of time (in miliseconds) the core is sleeping"
```

Minor nit, not blocking.

Perhaps shorten this to:
```suggestion
	int "Core sleep duration (miliseconds)"
```
after all this is a title and details are in the help text.

---

I wish to keep this as it is. It's already in several places:
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_ppr/Kconfig#L8
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_spim_loopback/Kconfig#L8
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_stm/Kconfig#L8
https://github.com/nrfconnect/sdk-nrf/blob/main/tests/benchmarks/multicore/idle_wdt/Kconfig#L8

---

OK, I will add commit that changes this KConfig title in all files.